source/crud/tests/v2/find-allowdiskuse-clientError.yml

```diff
@@ -1,11 +1,11 @@
 runOn:
-  - { maxServerVersion: "4.2" }
+  - { maxServerVersion: "3.0.99" }
 
-collection_name: &collection_name 'test_find_allowdiskuse_error'
+collection_name: &collection_name 'test_find_allowdiskuse_clienterror'
```

Comment:
Is this field even necessary if we're not expecting the collection name in an APM assertion?

Reply:
I wasn't sure if the test runners needed it for running the operation against _some_ collection. e.g. the `deleteOne-hint-clientError.yml` test specifies it despite not having any APM assertoins. I'm happy to remove it if test runners have a fallback collection they operate on; I just wasn't sure if that is the case.

Reply:
Feel free to leave as-is, then. All of the `v2` tests appear to have it even though the field is documented as optional. I can't speak for other drivers, but I definitely use my own default if this field in not available.